### DISCUSSION TITLE
Error: The domain is not available.

### DISCUSSION THREAD

**Discussion by ericzhu105** [vercel/community#106] [Help] [answered]
<img width="834" alt="Screen Shot 2021-12-18 at 8 32 22 PM" src="https://user-images.githubusercontent.com/69171189/146661816-0d041b18-9b84-4910-ab65-ae08aa3ba0a4.png">

On vercel I get this error when I try to add a domain

**Comment by leerob** [comment]
Is this a domain you're purchased through Vercel?

**Comment by den-sokolovus** [comment]
Hi Eric, first of all, some great news. On my end the domain is working fine with Vercel now. 
According to public Whois, the domain was edited with the Registrar on December 19, approximately the moment you posted the question. 
When domain nameservers are changed, it can take 24-48 hours for the changes to take effect. During that time the domain can appear as unavailable, up to the moment the new domain configuration is cached with various Internet providers, and it can be requested by the system for a check.
If my assumption is wrong, it would be great if you could share more details and/or a solution you've applied to fix the issue. We'll make sure it is noted down so we can help other users if they face a similar situation.
Thanks in advance!

**Comment by vojto** [comment, marked as answer]
I'm having the same issue:

<img width="801" alt="CleanShot 2022-01-02 at 15 11 51@2x" src="https://user-images.githubusercontent.com/50943/147878491-e45f0de4-a84f-4afb-bc2e-d960fa09539c.png">


The domain is registered using Namecheap, it's currently using their DNS servers, and pointed to another server. I haven't touched the domain config in a while.

I wanted to first add it to my project, and only then update the DNS configuration. It worked that way for all my other domains that I'm migrating now.

What am I missing?

**Reply by leerob** [reply]
Based on that error, that domain is likely attached to another project - do you own `tomato2.app`?

**Reply by vojto** [reply]
Yes - I already contacted support and it turns out it was attached to `super.so` - I tried the app ages ago and completely forgot.

So I have to remove it from super.so before adding to my own project. Thank you!

**Reply by leerob** [reply]
Awesome! Glad it's resolved 🙏
